pyOCD/gdbserver/gdbserver.py

```diff
@@ -332,21 +466,20 @@ def resume(self):
         self.timeOfLastPacket = time()
```

Comment:
Actually, changed my mind. I remove the time of last packet attributes. Also changed resume() to call wait() on the interrupt_event instead of sleeping.